src/components/Input/EditInput.tsx

```diff
@@ -0,0 +1,106 @@
+import type { InputProps, InputStyleOption, InputStylesheet } from './index'
+import type { ReactElement } from 'react'
+import styled from '@emotion/styled'
+import type { StyledProps } from '@types'
+
+type EditInputProps = Omit<InputProps, 'isPrice'>
+
+type EditStyleOption = 'WIDTH' | 'HEIGHT' | 'FONT' | 'PRICE_UNIT_BOTTOM'
+type EditStylesheet = InputStyleOption<EditStyleOption>
+
+interface StyledInputProps {
+  inputStylesheet: EditStylesheet
+}
+type StyledMessageProps = StyledProps<EditInputProps, 'status'>
+
+const EDIT_STYLESHEET: InputStylesheet<EditStylesheet> = {
+  large: {
+    FONT: 'display02M',
+    HEIGHT: 36,
+    PRICE_UNIT_BOTTOM: 16,
+    WIDTH: 714
+  },
+  small: {
+    FONT: 'body01R',
+    HEIGHT: 32,
+    PRICE_UNIT_BOTTOM: 14,
+    WIDTH: 360
+  }
+} as const
+
+export const EditInput = ({
+  inputSize = 'large',
+  label = '',
+  message = '',
+  status = 'default',
+  ...props
+}: EditInputProps): ReactElement => {
+  const hasMessage = status !== 'none'
+  const inputStylesheet = EDIT_STYLESHEET[inputSize]
+
+  return (
+    <StyledInputForm>
+      <StyledInputLabel>
+        {label}
+        <StyledInput inputStylesheet={inputStylesheet} {...props} />
```

Comment:
```suggestion
<StyledInput
  inputStylesheet={inputStylesheet}
  type="number"
  {...props}
/>
```
type 명시해주면 좋을 것 같습니다!

Reply:
[요기](dc4e0b948281b0d69dfbda51ee92ad4c4dbaad47)에 반영 완료 ! ☺️
위와 동일한 이슈로 적용하지 않았습니다!